pyvista/plotting/renderer.py

```diff
@@ -458,9 +458,11 @@ def enable_anti_aliasing(self):
 
         """
         if uses_egl():
-            warnings.warn(
-                "VTK compiled with OSMesa does not properly support anti-aliasing and anti-aliasing will not be enabled."
-            )
+            # only display the warning when not building documentation
+            if not pyvista.BUILDING_GALLERY:  # pragma: no cover
```

Comment:
Does it make sense to have the no-cover pragma here? The test suite doesn't have `pyvista.BUILDING_GALLERY`, does it? Would it make more sense to leave the whole `if uses_egl()` block uncovered?